docker/jenkins-images/Dockerfile-build-base

```diff
@@ -1,17 +1,20 @@
 ARG FROM_VERSION
 FROM harbor.h2o.ai/opsh2oai/h2o-3/dev-r-3.4.1:${FROM_VERSION}
 
-RUN \
-    curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
-    apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y \
-        texlive \
-        texlive-fonts-extra \
-        texlive-htmlxml \
```

Comment:
`texlive-htmlxml` doesn't exist anymore. In 18.04 it used to be a transitional package that installed

- [texlive-formats-extra (>= 2017.20170628)](https://answers.launchpad.net/ubuntu/bionic/amd64/texlive-formats-extra)
- [texlive-latex-extra (>= 2017.20170628)](https://answers.launchpad.net/ubuntu/bionic/amd64/texlive-latex-extra)
- [texlive-latex-recommended (>= 2017.20170628)](https://answers.launchpad.net/ubuntu/bionic/amd64/texlive-latex-recommended)
- [texlive-plain-generic (>= 2017.20170628)](https://answers.launchpad.net/ubuntu/bionic/amd64/texlive-plain-generic)

So I added those.

See https://answers.launchpad.net/ubuntu/bionic/amd64/texlive-htmlxml/2017.20180305-1